### PR TITLE
links, page header & people pic

### PR DESCRIPTION
**Pull request by marbaque**
- removed serif font from links class
- added padding on mobile to page header
- updated the people picture from the home page

### CODE REVIEW

**Comment by gandalfar** [comment]
Can you please merge upstream/master into your branch and fix conflicts?

File: resources/assets/styles/components/content-header.scss [hunk 1]
```diff
@@ -35,13 +35,14 @@ body.latam .contentheader-strip::after {
   left: 0;
   z-index: 10;
 
-  background-image: url("../images/patterns/pattern-pointy.svg"),
-    url("../images/patterns/pattern-circles.svg");
+  background-image: url("../images/patterns/pattern-pointy.svg"), url("../images/patterns/pattern-circles.svg");
```
Comment (gandalfar) [inline resources/assets/styles/components/content-header.scss:38]: is this just formatting change?

File: resources/views/components/content-header.blade.php [hunk 2]
```diff
@@ -1,92 +1,89 @@
 @if ($variant === 'sublanding')
-  <div class="contentheader-strip text-black2"
-       @if($background_image)style="background-image: url({!! $background_image !!}"@endif>
-    <div class="container flex flex-col justify-center">
-      <div class="content lg:px-6
+<div class="contentheader-strip text-black2" @if($background_image)style="background-image: url({!! $background_image !!}" @endif>
```
Comment (gandalfar) [inline resources/views/components/content-header.blade.php:2]: I don't understand what the non-formatting change is in this line and several lines below. Can you explain?

**Comment by gandalfar** [comment]
it's also not clear why all the formatting changes. If we need a better .editorrc, let me know but overall a PR shouldn't make indentation just changes without a good reason.

**Comment by marbaque** [comment]
Ok I think i was able to merge correctly after modifying [remote oeg]

**Pull request merged**